### PR TITLE
meta: remove nodejs/collaborators from codeowners

### PR DESCRIPTION
**Pull request by anonrig**
Mentioning nodejs/collaborators on every `/pages/en/guides` change is extremely frustrating for people who don't want to get notified, or has already a huge amount of notifications in their backlog. Rather than being `opt-out`, I **strongly** believe that this should be an `opt-in` thing. Basically, whenever the Node.js website team wants/interested in asking for opinion, they should request a review from the appropriate team, or depending on the severity from TSC.

cc @nodejs/tsc

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

**Review by ovflowd — APPROVED** [review]
I understand the sentiment and feel you. But once this gets merged, we'll have no way to ping Collaborators. We don't even know most of the time who should be pinged (We don't even know which teams to ping. Sometimes we do... but most of the times I'd say not 🤔)

I will not object to this PR, but I just wish that people felt interested in contributing to the content that technically represents the project.

**Review by AugustinMauroy — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]

**Review by benjamingr — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Comment by mhdawson** [comment]
@ovflowd we could create another team collaborators could opt into if they wanted to be notified. Not sure if that helps in terms of getting the views you like.

**Comment by ovflowd** [comment]
> @ovflowd we could create another team collaborators could opt into if they wanted to be notified. Not sure if that helps in terms of getting the views you like.

Hi Michael! Appreciate the gesture. Im just not sure how people would opt-in there and how we'd structure that. I feel that removing the current tag to collaborators is fine and we can always tag the TSC if there's content being edited and we're unsure who would be the right teams to review them.

**Comment by GeoffreyBooth** [comment]
> But once this gets merged, we'll have no way to ping Collaborators.

Couldn't you just manually tag `@nodejs/collaborators` in a PR description when desired? All this change does is preventing that from happening automatically.

**Comment by ovflowd** [comment]
> > But once this gets merged, we'll have no way to ping Collaborators.
> 
> Couldn't you just manually tag `@nodejs/collaborators` in a PR description when desired? All this change does is preventing that from happening automatically.

Right, I think what I meant is there's no default way to ping them besides explicitly pinging the whole team, but that is just the same as keeping in the codeowners.

**Pull request merged**
